### PR TITLE
fix: Menu drop to generate token on console.

### LINKED ISSUE #364: Feature: generate jwt for dev purposes
### Type of feature

🍕 Feature

### Current behavior

When testing the API, you need to console.log locally. What is we generate a token on demand for logged in users?

bonus: 
We could also lock it down to only appear for open-sauced org members.

## Step one - add a link to print JWT in the console.

Screen Shot 2022-10-06 at 7 22 00 AM

## Step two

Print the JWT to the console. 

### Additional context

https://github.com/open-sauced/api.opensauced.pizza/issues/48

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by janie-lee-developer**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

Adding an option at the drop down menu for the user to generate the bearer token on the console.

## Related Tickets & Documents

fix #364 

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by bdougie** [comment]
@open-sauced/triage

**Comment by shamimbinnur** [comment]
LGTM

**Review by 0-vortex — CHANGES_REQUESTED** [review]
Can you change the wording to "Print auth JWT"? 

We can merge shortly after that! 🍕 🍕 🍕

**Review by 0-vortex — APPROVED** [review]

File: src/components/PrimaryNav.tsx [hunk 1]
```diff
@@ -131,6 +132,19 @@ const PrimaryNav = (): JSX.Element => {
...
 
+                <Menu.Item>
+                  {({ active }) => (
+                    <button
+                      className={`${
+                        active ? "bg-gray-100 text-gray-700" : "text-gray-900"
+                      } group flex w-full items-center rounded-md px-5 py-1.5 text-sm`}
+                      onClick={() => console.log("Token: ", currentUser?.access_token)}
+                    >
+                      Print auth JWT
```
Comment (bdougie) [inline src/components/PrimaryNav.tsx:143]:
```suggestion
                      Print auth token
```
Think we should remove this acronym for clarity.

**Review by bdougie — CHANGES_REQUESTED** [review]
Just one small wording change requested.

**Pull request merged**